96-chromebook.install

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/bash
+
+COMMAND="$1"
+KERNEL_VERSION="$2"
+BOOT_DIR_ABS="$3"
+KERNEL_IMAGE="$4"
+
+if [[ -d "$BOOT_DIR_ABS" ]]; then
+    INITRD="initrd"
+else
+    BOOT_DIR_ABS="/boot"
+    INITRD="initramfs-${KERNEL_VERSION}.img"
+fi
+
+case "$COMMAND" in
+    add)
+        echo "Install required packages"
+        dnf install lz4 vboot-utils uboot-tools -y
+        if [[ -f "$BOOT_DIR_ABS/$INITRD" ]]; then
+            rm -f -- "$BOOT_DIR_ABS/$INITRD"
+        fi
+        dracut --force -v --force-drivers "ulpi usb-storage phy-qcom-usb-hs-28nm \
+        phy-qcom-usb-ss ocmem dwc3 dwc3-of-simple dwc3-pci ehci-platform xhci-plat-hcd \
+        i2c-qcom-geni i2c-qup icc-osm-l3 qcom-spmi-pmic phy-qcom-qmp phy-qcom-qusb2 \
+        phy-qcom-usb-hs qcom_aoss qcom-apcs-ipc-mailbox llcc-qcom nvmem_qfprom smem \
+        smp2p dwc3-qcom \
+        " "$BOOT_DIR_ABS/$INITRD" --kver "$KERNEL_VERSION" --kmoddir /lib/modules/"${KERNEL_VERSION}"
+
+        kernel="${KERNEL_IMAGE##*/}-${KERNEL_VERSION}.lz4"
+        compression="lz4"
```

Comment:
is this compression type required by Coreboot ?

Reply:
this is the compression method already being used by coreboot for the Initial kernel in the chromebook

Reply:
I see. Maybe worth adding a comment then?